src/Microsoft.Graph/Generated/requests/ConnectorMemberOfCollectionRequest.cs

```diff
@@ -16,17 +16,17 @@ namespace Microsoft.Graph
     using System.Linq.Expressions;
 
     /// <summary>
-    /// The type GraphServiceConnectorGroupsCollectionRequest.
+    /// The type ConnectorMemberOfCollectionRequest.
```

Comment:
removing an entityset from the entitycontainer, there should now be a containstarget = true for this, which there is.